packages/clerk-js/src/ui/components/SignUp/SignUpStart.tsx

```diff
@@ -210,7 +210,7 @@ function _SignUpStart(): JSX.Element {
     card.setError(undefined);
 
     const redirectUrl = buildSSOCallbackURL(ctx, displayConfig.signUpUrl);
-    const redirectUrlComplete = ctx.afterSignUpUrl || displayConfig.afterSignUpUrl;
+    const redirectUrlComplete = ctx.afterSignUpUrl || '/' || displayConfig.afterSignUpUrl;
```

Comment:
The final part of the clause here is never going to run.
```suggestion
    const redirectUrlComplete = ctx.afterSignUpUrl || '/';
```

Reply:
Done!